build.sh

```diff
@@ -6,12 +28,27 @@ export DO_UPDATE
 export VERBOSE
 export QUIET
 
+HTML_CACHE=${HTML_CACHE:-.cache}
```

Comment:
> Should we document these? [$HTML_CACHE, etc.] (Not sure myself.)

Me neither. So I suggest we just leave them undocumented for now unless/until we end up seeing evidence that the lack of documentation for them has caused any problems/confusion.